applications/mne_analyze/plugins/datamanager/FormFiles/datamanagerview.cpp

```diff
@@ -124,3 +128,18 @@ void DataManagerView::onNewFileLoaded(int iSubject,
     m_pUi->m_pTreeView->selectionModel()->select(m_pUi->m_pTreeView->model()->index(iModel, 0, m_pUi->m_pTreeView->model()->index(iSubject, 0)),
                                                  QItemSelectionModel::ClearAndSelect);
 }
+
+//=============================================================================================================
+
+void DataManagerView::keyPressEvent(QKeyEvent *event)
+{
+    switch (event->key())
+    {
+        case Qt::Key_Delete:
+            emit removeItem(m_pUi->m_pTreeView->currentIndex());
+            break;
```

Comment:
When hitting the delete key (or right clicking and selecting delete) on the subject itself, the dialog pops up as if a model were selected. Is this fine as is for now or should there be some check to make sure the item being deleted is a file to allow the drop down/pop up to appear?
![ts](https://user-images.githubusercontent.com/34070103/85346077-591dc600-b4c2-11ea-8852-3d6d4ba65c5c.png)

Reply:
good catch. I will fix this.